cmd/cli/namespace_add.go

```diff
@@ -102,9 +102,12 @@ func (a *namespaceAddCmd) run() error {
 	"metadata": {
 		"labels": {
 			"%s": "%s"
+		},
```

Comment:
Please modify this so that the annotation is only removed if `cmd.Flags().Changed("enable-sidecar-injection") && a.enableSidecarInjection == false`